doc/examples.rst

```diff
@@ -0,0 +1,12 @@
+.. _examples:
+
+Examples
+========
+
+To make use of Ibex it has to be integrated as described in :ref:`core-integration`.
+Here is an example of how this could look like.
+
+FPGA
+----
+
+A minimal example for the `Arty A7 <https://reference.digilentinc.com/reference/programmable-logic/arty-a7/start>`_ FPGA Development board is provided. Find the description of how to use this in ``examples/fpga/artya7-100/README.md``.
```

Comment:
I'd actually prefer having the documentation in the manual and link to it from the README, i.e. the other way around.

Reply:
So the description should move to doc, but the README will still hold the build instruction?